vertical-pod-autoscaler/docs/sidecar-containers.md

```diff
@@ -0,0 +1,173 @@
+# VPA Sidecar Container Management
+
+The Vertical Pod Autoscaler (VPA) has specific behavior when dealing with sidecar containers that are injected into pods via admission webhooks. This document explains the default behavior and how to customize it for your needs.
```

Comment:
Does "Sidecar" refer to "A container that isn't the main one", or to the formal Sidecar definition (ie: https://kubernetes.io/docs/concepts/workloads/pods/sidecar-containers/)

I assume it's the former. It may be good to clarity this.

Reply:
Added in [3e4addd](https://github.com/kubernetes/autoscaler/pull/7645/commits/3e4adddc3a8060f0821bf3825990d04fbc86c0b3)